cyences_app_for_splunk/default/data/ui/views/cs_asset_intelligence.xml

```diff
@@ -390,10 +390,11 @@
       <table>
         <title>Endpoint Service Not Running</title>
         <search>
-          <query>`cs_sophos` type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") | rename dhost as host | eval ip=src_ip | search $tkn_filter_main$ | stats count, latest(_time) as _time, values(src_ip) as src_ip, values(suser) as user, latest(type) as type by host
+          <query>index=main sourcetype="sophos*"  type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") host="*" location IN ("*") source IN ("*")
```

Comment:
Use the macro for the index and try to find the exact source type.
index=main sourcetype="sophos*" => `cs_sophos` sourcetype="sophos_events"

Do not required this filters => host="*" location IN ("*") source IN ("*") 
instead use this token for the filter => | search $tkn_filter_main$ (verify the field names in token value)